packages/core/strapi/bin/strapi.js

```diff
@@ -14,7 +14,12 @@ const inquirer = require('inquirer');
 const program = new Command();
 
 const packageJSON = require('../package.json');
-const { promptEncryptionKey, confirmMessage } = require('../lib/commands/utils/commander');
+const {
+  promptEncryptionKey,
+  confirmMessage,
+  parseURL,
+} = require('../lib/commands/utils/commander');
+const { ifOptions, assertUrlHasProtocol, exitWith } = require('../lib/commands/utils/helpers');
```

Comment:
> assertUrlHasProtocol

Do you think it makes sense to have this as a commander util? :thinking:

Reply:
Yeah, it would be better in a general Strapi utils if we have one. Let me know if you have a suggestion and I'll gladly move it :)

Reply:
I think for now it can stay here tbh.